docs/the-new-architecture/landing-page.md

```diff
@@ -1,32 +1,72 @@
+
+Beyond high-volume data processing with JSI, which many libraries have already adopted, JSI adoption in the New Architecture removes this class of serialization work from all native-JavaScript interop.
+
+### Learn more
+
+To achieve this, the New Architecture had to refactor multiple parts of the React Native infrastructure. To learn more about the refactor and other benefits it brings, check out the documentation in the New Architecture working group.
+
+## Benefits of the New Architecture for you
+
+While the New Architecture enables these features and improvements, using the New Architecture in your app or library does not mean guaranteed performance or user experience wins. Data serialization may not have been a bottleneck for your app's performance and certain features may require intentional adoption from your app or third-party dependencies to see benefit.
+
+Overall, enabling the New Architecture in your app or library is opting into the future of React Native as the framework and ecosystem leverage the new capabilities the New Architecture brings.
+
+## So, should I use the New Architecture today?
+
+Today, the New Architecture is considered experimental and we continue to refine backwards compatibility for a better adoption experience.
+
+The team plans to enable the New Architecture by default in an upcoming React Native release by the end of 2024. For most React Native developers, we recommend waiting until that release before enabling the New Architecture.
+
+Our guidance is as follows
+
+- For most production apps, we do _not_ recommend enabling the New Architecture at this time. Waiting for the official release will offer the best experience.
+- If you maintain a React Native library, we recommend enabling it and verifying your use-cases are covered. You can find the [instructions here](https://github.com/reactwg/react-native-new-architecture/blob/lunaleaps-move-docs-over/README.md#documentation).
+
+Due to the major refactor, certain conventions and APIs are no longer supported in the New Architecture. To improve the adoption experience, the team is currently working with library authors and early adopters to catch these use-cases and provide suitable interop between the legacy and New Architecture.
+
+If you are interested in dogfooding the New Architecture experience, you can find [instructions](https://github.com/reactwg/react-native-new-architecture/blob/lunaleaps-move-docs-over/README.md#documentation) in our dedicated working group. The [New Architecture working group](https://github.com/reactwg/react-native-new-architecture) is a dedicated space for support and coordination for New Architecture adoption and where the team posts regular updates.
```

Comment:
This link lands to a Table of Content. I'd say we would need more of a "How to enable the New Architecture" kind of page on the WG end

Reply:
I need to actually move those docs over and will update the link to the actual instructions